bundles/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/NetworkHandlerConfiguration.java

```diff
@@ -31,5 +31,6 @@ public class NetworkHandlerConfiguration {
     public Integer retry = 1;
     public Integer refreshInterval = 60000;
     public Integer timeout = 5000;
+    public boolean assumeiOsDevice = true;
```

Comment:
I would suggest rather to default this to false for NEW things.
Please see the reasoning here: https://github.com/openhab/openhab-addons/issues/9575#issuecomment-1886812359
But understand for EXISTING devices it is safer to default to true -> if parameter is not defined assume value as true.
Maybe consider to apply [Thing Upgrades](https://www.openhab.org/blog/2023-07-23-openhab-4-0-release.html#thing-upgrades) while Openhab upgrade.

Reply:
No, changing the default behaviour is a breaking change. The Thing upgrade does not affect configuration.